src/components/Homepage/HomepageStatistics.vue

```diff
@@ -0,0 +1,205 @@
+<template>
+	<section class="statistics section">
+		<div class="row">
+			<h2
+				v-html="statsHeadline"
+				class="small-12 large-6 columns statistics__header green-emphasis"
+			>
+			</h2>
+			<span class="statistics__video-wrapper small-12 large-6 columns">
+				<video
+					:src="statsVideo.file.url"
+					class="statistics__video-wrapper--video"
+					autoplay
+					loop
+					muted
+					playsinline
+				></video>
+			</span>
+		</div>
+		<div class="row stat-block">
+			<div
+				v-for="(statBlock, index) in statsBlockText"
+				:key="statBlock.key"
+				class="small-12 medium-4 columns"
+			>
+				<kv-contentful-img
+					v-if="statBlock.image.url"
+					:class="'stat-block__icon stat-block__icon-' + index"
+					:contentful-src="statBlock.image.url"
+					:alt="statBlock.image.description"
+					fallback-format="png"
+					:width="48"
+					:height="48"
+				/>
+				<p
+					class="stat-block__stat green-emphasis"
+					v-html="formattedTextStrings[index]"
+				>
+				</p>
+			</div>
+		</div>
+	</section>
+</template>
+
+<script>
+import _get from 'lodash/get';
+import numeral from 'numeral';
+import KvContentfulImg from '@/components/Kv/KvContentfulImg';
+import whyKivaQuery from '@/graphql/query/whyKivaData.graphql';
+import { buildDynamicString } from '@/util/contentfulUtils';
+import { documentToHtmlString } from '~/@contentful/rich-text-html-renderer';
+
+export default {
+	inject: ['apollo', 'cookieStore'],
+	data() {
+		return {
+			repaymentRate: '',
+			totalLoansInDollars: '',
+			numCountries: '',
+			numLenders: '',
+		};
+	},
+	apollo: {
+		query: whyKivaQuery,
+		preFetch: false,
+		result({ data }) {
+			this.repaymentRate = _get(data, 'general.kivaStats.repaymentRate');
+			this.totalLoansInDollars = _get(data, 'general.kivaStats.amountFunded');
+			this.numCountries = _get(data, 'general.kivaStats.numCountries');
+			this.numLenders = _get(data, 'general.kivaStats.numLenders');
+		}
+	},
+	components: {
+		KvContentfulImg,
+	},
+	props: {
+		content: {
+			type: Object,
+			default: () => {},
+		},
+	},
+	computed: {
+		statsText() {
+			return this.content?.contents?.find(({ key }) => key.indexOf('homepage-statistics-headline-text') > -1);
+		},
+		statsHeadline() {
+			return this.statsText.headline;
```

Comment:
this.statsText might be undefined, so I would recommend using `this.statsText?.headline ?? ''`